build

```diff
@@ -65,18 +65,18 @@ make_relup() {
     if [ -d "$releases_dir" ]; then
         while read -r zip; do
             local base_vsn
-            base_vsn="$(echo "$zip" | grep -oE "[0-9]+\.[0-9]+\.[0-9]+(-[0-9a-f]{8})?")"
+            base_vsn="$(echo "$zip" | grep -oE "[0-9]+\.[0-9]+\.[0-9]+(-[0-9a-f]{8})?" | head -1)"
             if [ ! -d "$releases_dir/$base_vsn" ]; then
                 local tmp_dir
                 tmp_dir="$(mktemp -d -t emqx.XXXXXXX)"
                 unzip -q "$zip" "emqx/releases/*" -d "$tmp_dir"
                 unzip -q "$zip" "emqx/lib/*" -d "$tmp_dir"
-                cp -r -n "$tmp_dir/emqx/releases"/* "$releases_dir"
-                cp -r -n "$tmp_dir/emqx/lib"/* "$lib_dir"
+                cp -r -n "$tmp_dir/emqx/releases"/* "$releases_dir" || true
+                cp -r -n "$tmp_dir/emqx/lib"/* "$lib_dir" || true
                 rm -rf "$tmp_dir"
             fi
             releases+=( "$base_vsn" )
-        done < <(find _upgrade_base -maxdepth 1 -name "*$PROFILE-otp${OTP_VSN}-$SYSTEM*-$ARCH.zip" -type f)
+        done < <(find _upgrade_base -maxdepth 1 -name "*$PROFILE-*-otp${OTP_VSN}-$SYSTEM*-$ARCH.zip" -type f)
```

Comment:
Why `$PROFILE-*-otp${OTP_VSN}` ?

Reply:
I don't know what is the "correct" format of the filename, but after running `make <profile>-zip` I got `emqx-4.4-beta.1-0c5ec9f2-otp23.2.5-macos10-amd64.zip` using 4.4.0 opensource and `emqx-ee-4.4.0-otp23.2.5-macos10-amd64.zip` or `emqx-ee-4.4.0-6d388f8b-otp23.2.5-macos10-amd64.zip` using 4.4.0 enterprise.

Reply:
`$PROFILE` is defined in the `Makefile`, in the open source version it is defined in `. /Makefile`, in the Enterprise version it is in `. /enterprise.mk`

Reply:
I see, I misunderstood, the "-*-" is to match the version number, it's looks good for me

Reply:
Wait, I think `*$PROFILE-otp${OTP_VSN}-$SYSTEM*-$ARCH.zip` should be changed to `*$PROFILE-*-otp${OTP_VSN}-$SYSTEM-$ARCH.zip`, this is a more accurate match for the 4.4 package name